style.css

```diff
@@ -0,0 +1,129 @@
+body {
+  width: auto;
+  margin: 0;
+  height: 100vh;
+}
+
+a {
+  color: inherit;
+}
+
+.top-nav {
+  padding-left: 20px;
+  padding-right: 20px;
+  text-align: center;
+}
+
+.top-nav ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  height: 44px;
+  color: #28352f;
+}
+
+.top-nav span { padding-right: 10px; }
+
+.time {
+  padding-left: 30px;
+  font-style: normal;
+  font-weight: 500;
+  font-family: 'Inter', sans-serif;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  letter-spacing: -0.28px;
+}
+
+.menu-nav {
+  padding-top: 5px;
+  padding-bottom: 5px;
+  text-align: center;
+  height: 50px;
+  font-size: 30px;
+}
+
+.menu-nav ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0;
+  font-style: normal;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+  align-items: center;
+  color: #28352f;
+}
+
+.menu-icon {
+  font-size: 30px;
+  padding-right: 50px;
+}
+
+.name {
+  padding-left: 30px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 16px;
+}
+
+.headline {
+  background: url("assets/images/Header-llustration-desktop.svg");
```

Comment:
[OPTIONAL] Awesome work on setting the background with `CSS` property :clap: :+1: . However as you may have noticed, the background image is not adjusting to the screen size:

![Screenshot from 2022-02-11 04-49-05](https://user-images.githubusercontent.com/67608876/153516325-41571388-6470-46a6-b702-86bc77b4bef8.png)

One of the simple options to handle this is to add a `background-size ` property with a value of `contain`.  And please set the value of the ` background-repeat` to `no-repeat` otherwise we will get :infinity:  repeats of the bg image :relaxed: 
Sample code:

```CSS
.headline {
    background: url("assets/images/Header-llustration-desktop.svg");
    background-size: contain;
    background-repeat: no-repeat;
    padding-left: 24px;
    padding-right: 24px;
}
````
More on that please check [here](https://developer.mozilla.org/en-US/docs/Web/CSS/background-size)